.gitignore

```diff
@@ -16,3 +16,4 @@ data
 *.bak
 *~
 *.vscode
+.continuous-integration/tmp/
```

Comment:
Why is your editor doing this?

Reply:
I added this myself since it risks uploading the tmp folder if someone runs the checks locally.

Reply:
I meant the fact that there is now no newline at the end of the file.  Actually, the directory is guaranteed to be deleted by the script.  Did that not happen?

Reply:
If the script fails for some reason the directory is not deleted.

Reply:
Ah, the call to read is not in the try block.  Can you move it there?  Then deletion is guaranteed.

Reply:
I could go either way.  The script to generate the output should never fail.

Reply:
see daf3038. I think it doesn't hurt to leave it here